department-of-veterans-affairs/caseflow
# Queue | Connect judge checkout to dispatch FE to BE

Resolves #5152, resolves #5153

### Description
Connects FE & BE for Judge checkout to Dispatch flows

### Acceptance Criteria 
- [ ] Ready for Dispatch and Assign OMO flows work as expected

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE` / a judge
2. All of `BVAAABSHIRE`'s default cases are draft decisions: Go to case detail view, select Ready for Dispatch, complete flow (review dispositions, review remand reasons if applicable, evaluate decision), confirm you're navigated back to `/queue` with success message
3. To test OMO flow, switch users to attorney (`BVASCASPER1`) and complete Medical Request Ready for Review checkout flow, assigning to Anjali Abshire
4. Switch back to `BVAAABSHIRE`, navigate to case detail view for OMO case, choose Assign OMO from dropdown, confirm you're navigated back to `/queue` with success message.

### TODO
- Enable/disable judge checkout dropdown options based on document id / decass deprod attr #5998 

<details><summary>Assign to OMO</summary>
<img src="https://user-images.githubusercontent.com/8533004/41933707-a4383740-7952-11e8-9a50-b3040bdd5ac6.gif">
</details>

<details><summary>Ready for Dispatch</summary>
<img src="https://user-images.githubusercontent.com/8533004/41933719-ab5ed13c-7952-11e8-80bd-dd3ba85995e5.gif">
</details>
